app/models/dfe_signin/authorisation.rb

```diff
@@ -0,0 +1,26 @@
+module DfeSignin
```

Comment:
What do you think about getting rid of this class altogether and having the `has_role?` method directly on the `DfeSignin::UserAccess` class instead? From what I can see, the `DfeSignin::UserAccess` object has everything it needs to be able to say whether it has a role or not.

Reply:
Yeah, that's probably a good call actually. I only had this model when we had the old `authorised?` method, which was application-specific, but given `has_role?` can be used for any role, it's probably better to sit in the lib code